lib/plug/static.ex

```diff
@@ -346,8 +348,14 @@ defmodule Plug.Static do
         file_info(size: size, mtime: mtime) = file_info
         {size, mtime} |> :erlang.phash2() |> Integer.to_string(16)
     end
+    |> maybe_add_quotes()
```

Comment:
Instead of always adding it, I would say that if you have a custom etag function, then you are responsible for adding the quotes. We can actually improve the docs here. Then we make sure that our code is always quoted.

Reply:
Yes, but I wanted this version to work correctly with any existing etag generators, regardless of whether they use quotes or not. But of course this could be change in the future if the documentation states clearly that custom generators should produce quoted values.